### PR TITLE
Add note about --ignore on globbing docs page

### PR DESCRIPTION
**Pull request by underyx**

### CODE REVIEW

File: docs/Users/Glob_Patterns.rst [hunk 1]
```diff
@@ -37,6 +37,11 @@ The special characters used in shell-style wildcards are:
 | ``**``            | Matches everything.                                     |
 +-------------------+---------------------------------------------------------+
 
+.. note::
+
+    If you're looking for a negation pattern to ignore files, check out the
```
Comment (sils1297) [inline docs/Users/Glob_Patterns.rst]: to ignore files -> to exclude files

people may look for `exclude` specifically so that keyword might trigger it

Comment (sils1297) [inline docs/Users/Glob_Patterns.rst]: also I think I'd just remove the `negation` of that sentence

Comment (underyx) [inline docs/Users/Glob_Patterns.rst]: Decided against removing 'negation' just to keep another Ctrl-F keyword here.

**Comment by sils1297** [comment]
https://github.com/coala-analyzer/coala/pull/2251/commits/7efa246c278e764c94ee35884b796ada45e129af is ready

**Comment by sils1297** [comment]
reack https://github.com/coala-analyzer/coala/pull/2251/commits/16d3f680216497cd6abc28e42a5958b63b716876

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**